src/services/video-conference-manager/index.ts

```diff
@@ -310,6 +312,10 @@ export default class VideoConfereceManager {
     let frameWidth: string = `${width}px`;
     let frameHeight: string = `${height + waterMarkHeight}px`;
 
+    if (this.frameConfig.centralizeMeetingModals) {
+      frameWidth = '100%';
+    }
```

Comment:
This shouldn't be here. Because it affects the dynamic frame calculation. Would be better if it was in the inside the meeting frame